Bindings/Java/Matlab/Utilities/osimList2MatlabCell.m

```diff
@@ -0,0 +1,73 @@
+function matlabcellarray = osimList2MatlabCell(model, classname)
+%% osimList2MatlabCell()
+%   Convert an OpenSim list into a cell array of OpenSim
+%   references. The cell array can then be used to get
+%   references to individual components. 
+%
+% 
+%   % Get a cell array of references to all the bodies in a model
+%   references = osimList2MatlabCell(model,'Body')
+%   % Get the first body in the list.  
+%   Pelvis = references{1}
+%
+%    
+%   When iterating through a list to get/set properties on multiple 
+%   components, use the list iterator directly;
+%
+%   % Set a value for a property for all of the components of type 'Body'
+%   list = model.getBodyList();
+%   li = list.begin()
+%   while ~li.equals(list.end())
+%       li.next();
+%       li.setSomePropertyValue(val);
+%   end
```

Comment:
I think people can use the cell array even in this case. If you want to leave this documentation here anyway, perhaps motivate it instead by saying "Here's what this function does internally."

Reply:
Thanks for the note— I left the code snippet as an example and explained what is being done internally.